### ISSUE TITLE
QUESTION: expectations

### ISSUE THREAD

**Issue by fede72bari** [DHDev0/Muzero-unplugged#2]
I am a beginner and I am trying different RL algorithms on CartPole game to evaluate the performance. I run Muzero-unplugged example on CartPole with the proposed experimental settings and with slightly different ones, decreasing the learning rate, defining 2 hidden layers and extending the max steps per episode. Running the training example on the command line can see that after 1050 epochs (episodes?) the reward is still between 10 and 40, it appears more random than something that has been learned. So the question is: what could be the expected learning curve?
